weekly-updates/next.md

```diff
@@ -63,7 +63,8 @@ This week there was a new app:
 * 
 
 There have also been updates to: 
-* 
+
+* [Solid Focus](https://github.com/NoelDeMartin/solid-focus), a Task Manager for Solid. The latest update includes editing, scheduling, descriptions and deleting tasks.
```

Comment:
`descriptions and`
-> `describing, and`

Reply:
@TallTed Done.